packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/expiration-modal.tsx

```diff
@@ -2,14 +2,13 @@ import React from 'react';
 import { Button, Div100vhContainer, Modal, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import Expiration from './expiration';
-import { observer } from '@deriv/stores';
 
 type TMultipliersExpirationModalProps = {
     is_open: boolean;
     toggleModal: () => void;
 };
 
-const MultipliersExpirationModal = observer(({ is_open, toggleModal }: TMultipliersExpirationModalProps) => (
+const MultipliersExpirationModal = ({ is_open, toggleModal }: TMultipliersExpirationModalProps) => (
```

Comment:
During TS migration (6th branch) I forgot to remove observer component after removing unused variables from store.